docs/source/testing.rst

```diff
@@ -0,0 +1,12 @@
+torch.testing
+=============
+
+.. warning::
+
+    This module is in BETA. New functions are still being added, and some functions may change in future PyTorch
+    releases. See the documentation of each function for details.
```

Comment:
@mruberry Since we opted for releasing in a prototype state instead, any recommendations on the wording?

```suggestion
    This module is in a PROTOTYPE state. New functions are still being added, and the available functions may change in future PyTorch releases. We are actively looking for feedback for UI/UX improvements or missing functionalities.
```

Reply:
Let's just wait on this part of the stack until we get more feedback.